src/main/java/database/entity/Event.java

```diff
@@ -0,0 +1,151 @@
+package database.entity;
+
+import java.util.Date;
+import java.util.List;
+
+public class Event {
+    private String title;
+    private User owner;
+    private List<User> participants;
+    private String description;
+    private String photo;
+    private Long latitude;
+    private Long longitude;
+    private Date startDate;
+    private Date endDate;
+    private boolean showGuestList;
+    private int maxParticipants;
+    private boolean onlyRegistered;
+    private List<Category> categories;
```

Comment:
jeden event moze byc do kilku kategorii? Nie lepiej do jednej?

Reply:
brzmi dobrze, zmienie to